tree/dataframe/inc/ROOT/RDF/ActionHelpers.hxx

```diff
@@ -524,6 +634,34 @@ public:
 
    void Finalize()
    {
+#ifdef RH_CUDA
+      if (getenv("CUDA_HIST")) {
```

Comment:
Maybe `ROOT_RDF_CUDA`, given that we want to expand this and that it's ROOT specific?